ads/_prefetch.js

```diff
@@ -39,5 +39,6 @@ export const adPreconnect = {
   doubleclick: [
     'https://partner.googleadservices.com',
     'https://securepubads.g.doubleclick.net',
+    'https://tpc.googlesyndication.com',
```

Comment:
Just one clarification. Did we drop our rule to not have trailing comes in arrays?